### PR TITLE
Remove paddings from card-container

### PR DESCRIPTION
**Pull request by annano**

### CODE REVIEW

File: plugins/recruit-resources/src/components/KanbanCard.svelte [hunk 1]
```diff
@@ -28,51 +28,56 @@
   export let object: WithLookup<Applicant>
   export let dragged: boolean
 
-  function showCandidate () {
+  function showCandidate() {
```
Comment (haiodo) [inline plugins/recruit-resources/src/components/KanbanCard.svelte]: Please use eslint

File: packages/kanban/src/components/Kanban.svelte [hunk 2]
```diff
@@ -261,9 +257,7 @@
       cursor: grab;
     }
     &.dragged {
-      padding: 1rem;
```
Comment (haiodo) [inline packages/kanban/src/components/Kanban.svelte:264]: Why? this was done by purpose, to make all of them look similar during dragging.

Comment (annano) [reply packages/kanban/src/components/Kanban.svelte:264]: I think this padding is to compensate the border, which I have also removed from class, the border was the same color as a card-container or unset, so I think we can just remove it at all, and without the border this padding is also not needed.

**Review by haiodo — APPROVED** [review]

File: plugins/recruit-resources/src/components/KanbanCard.svelte (hunk 1 above)
Comment (annano) [reply plugins/recruit-resources/src/components/KanbanCard.svelte]: Done

**Review by haiodo — APPROVED** [review]

**Pull request merged**